module-4/backend/week_one.md

```diff
@@ -5,18 +5,32 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's the most useful thing you learned from completing the intermission week work?
+- How to javascript in general
 2. What are some tools to help debug JavaScript code?
+- debugger/chrome devtools
 3. What are some tools you need in order to unit test your JavaScript?
+- mocha/chai
 4. What is the syntax for invoking a function?
+- functionName()
 5. What is CORS?
+- Cross-Origin Resource Sharing -- basically securing cross domain resource sharing and blocking the API calls unless specifically allowed
 6. How do we enable CORS?
+- CORS is enabled by default in modern browsers
 7. What's `this` in JavaScript?
+- this refers to the function that is currently being processed.
```

Comment:
The value of `this` changes on the context it is used in. In the global scope, it refers to your browser's window. In a constructor function, it would refer to the object declared, etc.